vue/components/forms/personalization-form/reference.vue

```diff
@@ -180,6 +181,16 @@ export const Reference = {
         if (this.onPostConfig) this.$bus.unbind(this.onPostConfig);
     },
     methods: {
+        hasValidInput(initials) {
+            if (!initials) return true;
+
+            const validChars = !initials
+                .split("")
+                .find(char => !this.$store.state.initialsSupportedCharacters.find(c => c === char));

```

Comment:
Try to avoid the double negative :p. Also, you can: `const validChars = initials.every(initial => this.$store.state.initialsSupportedCharacters.includes(initial)`